apps/speed.c

```diff
@@ -3736,12 +3746,14 @@ static void multiblock_speed(const EVP_CIPHER *evp_cipher, int lengths_single,
                 aad[12] = (unsigned char)(len);
                 pad = EVP_CIPHER_CTX_ctrl(ctx, EVP_CTRL_AEAD_TLS1_AAD,
                                           EVP_AEAD_TLS1_AAD_LEN, aad);
-                EVP_Cipher(ctx, out, inp, len + pad);
+                ciph_success = EVP_Cipher(ctx, out, inp, len + pad);
             }
         }
         d = Time_F(STOP);
         BIO_printf(bio_err, mr ? "+R:%d:%s:%f\n"
                    : "%d %s's in %.2fs\n", count, "evp", d);
+        if (ciph_success <= 0)
```

Comment:
better not output/print when `mr` flag is true.
`mr` signals that fork mode is used, and that results are returned to parent process in a particular format, in order to be merged and reported ...

Just give a try to see what happens when speed is launch with fork mode enabled

Reply:
Fair point, thanks! Fixup pushed